src/services/iasql.ts

```diff
@@ -231,3 +233,99 @@ export async function dump(dbId: string, dataOnly: boolean) {
     await conn2?.close();
   }
 }*/
+
+export async function upgrade() {
+  const dbs = readdirSync('/tmp/upgrade');
+  const dbsDone: { [key: string]: boolean } = {};
+  for (const db of dbs) {
+    logger.info(`Starting Part 2 of upgrading ${db}`);
```

Comment:
Just for consistency 
```suggestion
    logger.info(`Starting Part 2 of 3 for ${db}`);
```